mergebot_backend/github_helper.py

```diff
@@ -13,6 +13,20 @@
 BOT_NAME = 'asfgit'
 GITHUB_SECRET = 'mergebot.secret'
 
+COMMENT_PAYLOAD = '{{"body": "{body}"}}'
+COMMIT_STATUS_PAYLOAD = """
+{{
+  "state": "{state}",
+  "target_url": "{url}",
+  "description": "{description}",
+  "context": "{context}"
+}}
+"""
+COMMIT_STATE_PENDING = 'pending'
+COMMIT_STATE_SUCCESS = 'success'
+COMMIT_STATE_ERROR = 'error'
+COMMIT_STATE_FAILURE = 'failure'
```

Comment:
Error is a problem that happened on MergeBot's side -- something that should be retried. Failure indicates something went wrong that wasn't MergeBot's fault -- bad rebase, tests failed, merge conflict, etc.